views/index.erb

```diff
@@ -156,17 +141,6 @@
                     Simulate CSP Error
                   </label>
                 </div>
-                <div class="usa-checkbox">
-                  <input class="usa-checkbox__input"
-                         id="check_enable_attempts_api"
-                         type="checkbox"
-                         name="enable_attempts_api"
```

Comment:
Lingering references with this input as well:

https://github.com/18F/identity-oidc-sinatra/blob/48344056a069ba99f4cc426d0356701f208498af/app.rb#L71-L73

https://github.com/18F/identity-oidc-sinatra/blob/48344056a069ba99f4cc426d0356701f208498af/app.rb#L81

https://github.com/18F/identity-oidc-sinatra/blob/48344056a069ba99f4cc426d0356701f208498af/app.rb#L153-L155

https://github.com/18F/identity-oidc-sinatra/blob/48344056a069ba99f4cc426d0356701f208498af/app.rb#L166

https://github.com/18F/identity-oidc-sinatra/blob/48344056a069ba99f4cc426d0356701f208498af/spec/app_spec.rb#L177-L184

https://github.com/18F/identity-oidc-sinatra/blob/48344056a069ba99f4cc426d0356701f208498af/spec/app_spec.rb#L266-L275